src/FirstSteps/RPG/Heroes/Hero.cs

```diff
@@ -7,20 +7,19 @@ namespace FirstSteps.RPG.Heroes
 {
     public abstract class Hero
     {
-        private List<Item> backPack = new List<Item>();
-        private int _coinsBag;
+        public Equipment equipment { get { return _equipment; } }
         private string _name;
         private Races _race;
         protected int _strength;
         protected int _intelligence;
         protected int _agility;
         protected int _health;
         protected int _damage;
-        private int maxBackpack = 5;
-        private int _maxWeight = 40;
-        public int Coins { get { return _coinsBag; } }
```

Comment:
Probably `Coins` property should stay and return value from the equipment. Look at the `MineTests.cs` file, now `dwarf.Coins.Should().Be(1);` this line of code does not work.